torchmetrics/metric.py

```diff
@@ -262,7 +262,7 @@ def sync(
             cache: A dictionary containing the local metric states. The cache will be empty if sync didn't happen.
         """
         is_distributed = distributed_available() if callable(distributed_available) else None
-        if not should_sync or not is_distributed:
+        if not should_sync or not is_distributed or not self._update_called:
```

Comment:
This is getting less and less obvious.

I'm assuming the issue is that syncing empty lists fails, maybe we should fix it on the syncing side instead?

Reply:
yes exactly, sync is done on empty states